widgets/filters/src/FilterActions.ts

```diff
@@ -0,0 +1,10 @@
+import { Filter } from './types';
```

Comment:
Optional:

`import type` would save webpack some work :slightly_smiling_face: 

```suggestion
import type { Filter } from './types';
```